help/bug_report.md

```diff
@@ -4,13 +4,20 @@ title: Bug Reports
 permalink: /bug_report/
 ---
 
-* <a href="https://github.com/triplea-game/triplea/issues?q=is%3Aissue+is%3Aopen+label%3ABug">Open Bug List</a>
+* [Open Bug List](https://github.com/triplea-game/triplea/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A%20bug%22)
 
-## Filing a bug report:
+## Before doing anything
 
-* Create a <a href="https://github.com/account"> github account</a>
-* Submit a tracking issue: <a href="https://github.com/triplea-game/triplea/issues/new">https://github.com/triplea-game/triplea/issues/new</a>, you'll need to give the topic a title, and fill out a description of what you would like done.  Please try to be as concise and specific as you can. Always please try to give specific examples of problems, map names, etc, to help make it very easy to focus in directly on what the problem is. 
+Before submitting a bug report at all, take a step back and make sure it's really a bug and not a feature request.
+Feature Request often need to be discussed first, if you still want to submit your feature Request, please do so in [our Forum](https://forums.triplea-game.org/category/42/feature-requests-ideas).
```

Comment:
Recommend striking this next sentence, instead create a section to capture this content, something like:

```
"## Send us Suggestions"

* What would you like to see improved? You can let us know here *link*
```

The reason I suggestion this is for skimming value, assume someone just had a game crash, they lost 5 hours of play and an interesting game, we are lucky almost at all if they will give us one more click. By that token, we need directions that are really dead simple, here is how you submit a bug report, one two three.